### PR TITLE
Update chart to latest

### PR DESCRIPTION
**Pull request by MarcelMue**
### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: helm/kyverno/values.yaml [hunk 1]
```diff
@@ -42,23 +51,33 @@ podSecurityContext: {}
...
 # dnsPolicy determines the manner in which DNS resolution happens in the cluster
 # in case of hostNetwork: true, usually, the dnsPolicy is suitable to be "ClusterFirstWithHostNet"
 # for further reference: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
 dnsPolicy: "ClusterFirst"
 envVars: {}
 extraArgs: []
-# - --webhooktimeout=4
 resources:
   limits:
-    memory: 1024Mi
```
Comment (stone-z) [inline helm/kyverno/values.yaml:58]: Down?

File: helm/kyverno/templates/_helpers.tpl [hunk 2]
```diff
@@ -28,35 +28,18 @@ If release name contains chart name it will be used as a full name.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
-{{/*
-Common labels
-*/}}
-{{- define "labels.common" -}}
-{{ include "labels.selector" . }}
```
[the remark this answers is not in the document]
Comment (MarcelMue) [reply helm/kyverno/templates/_helpers.tpl:35]: Yes - I think we used our default helpers so far. These are the upstream helpers. It seemed better to be as close to upstream as possible.

File: helm/kyverno/templates/clusterrole.yaml [hunk 3]
```diff
@@ -158,9 +182,10 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  labels:
-    rbac.authorization.k8s.io/aggregate-to-admin: "true"
   name: {{ template "kyverno.fullname" . }}:admin-policies
+  labels: {{ include "kyverno.labels" . | nindent 4 }}
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
```
[the remark this answers is not in the document]
Comment (MarcelMue) [reply helm/kyverno/templates/clusterrole.yaml:187]: This was here before already - so not entirely sure :D

File: helm/kyverno/values.yaml (hunk 1 above)
Comment (MarcelMue) [reply helm/kyverno/values.yaml:58]: Yes - will fix changelog. We discussed.

**Review by stone-z — APPROVED** [review]
IIWFYIWFM

**Pull request merged**